Cannot build on fresh Angular 16 app

Reproduction:
https://github.com/cg-roling/store-persist-repro-16

I created a new Angular 16 app with `npx @angular/cli@16 new`, installed `ngrx/store` and `ngrx-addons/persist-state`, set up a very basic reducer, and the app fails to serve with typings errors:

```
Error: node_modules/@ngrx-addons/persist-state/lib/persist-state.feature.module.d.ts:5:42 - error TS2314: Generic type 'PersistStateFeatureModule<T>' requires 1 type argument(s).

5     static ɵfac: i0.ɵɵFactoryDeclaration<PersistStateFeatureModule, never>;
                                           ~~~~~~~~~~~~~~~~~~~~~~~~~


Error: node_modules/@ngrx-addons/persist-state/lib/persist-state.feature.module.d.ts:6:43 - error TS2314: Generic type 'PersistStateFeatureModule<T>' requires 1 type argument(s).

6     static ɵmod: i0.ɵɵNgModuleDeclaration<PersistStateFeatureModule, never, never, never>;
                                            ~~~~~~~~~~~~~~~~~~~~~~~~~


Error: node_modules/@ngrx-addons/persist-state/lib/persist-state.feature.module.d.ts:7:43 - error TS2314: Generic type 'PersistStateFeatureModule<T>' requires 1 type argument(s).

7     static ɵinj: i0.ɵɵInjectorDeclaration<PersistStateFeatureModule>;
```

I've also tried with multiple combinations of Angular version (14/15/16) and persist-state version (1.2, 2.0, 

Strangely, it **does** work on this stackblitz, with Angular 15, and I'm unable to figure out what the difference is:
https://stackblitz.com/edit/angular-kpagc2?file=package.json

--- a/libs/persist-state/src/lib/persist-state.feature.module.ts
+++ b/libs/persist-state/src/lib/persist-state.feature.module.ts
@@ -2,7 +2,7 @@ import { NgModule } from '@angular/core';
 import { PersistStateFeature } from './persist-state.feature';
 
 @NgModule()
-export class PersistStateFeatureModule<T> {
+export class PersistStateFeatureModule<T = object> {
   constructor(state: PersistStateFeature<T>) {
     state.addFeature();
   }

--- a/libs/sync-state/src/lib/sync-state.feature.module.ts
+++ b/libs/sync-state/src/lib/sync-state.feature.module.ts
@@ -2,7 +2,7 @@ import { NgModule } from '@angular/core';
 import { SyncStateFeature } from './sync-state.feature';
 
 @NgModule()
-export class SyncStateFeatureModule<T> {
+export class SyncStateFeatureModule<T = object> {
   constructor(state: SyncStateFeature<T>) {
     state.addFeature();
   }

--- a/tsconfig.base.json
+++ b/tsconfig.base.json
@@ -11,8 +11,6 @@
     "target": "es2015",
     "module": "esnext",
     "lib": ["es2019", "dom"],
-    "skipLibCheck": true,
-    "skipDefaultLibCheck": true,
     "baseUrl": ".",
     "paths": {
       "@ngrx-addons/common": ["libs/common/src/index.ts"],

